packages/commonwealth/client/scripts/views/components/component_kit/new_designs/cw_button.tsx

```diff
@@ -15,10 +15,13 @@ type ButtonHeight = 'lg' | 'med' | 'sm';
 
 type ButtonWidth = 'narrow' | 'wide' | 'full';
 
+type ButtonAlt = 'alt-green' | 'alt-rorange';
```

Comment:
can we do just `green`, `rorage`, without `alt` prefix ?

Reply:
@masvelio 
I see what you mean, given that the 'alt' is contained in the property name 'ButtonAlt'. I added the 'alt' as a way to avoid class name clashes in the future, but given the specificity of the CSS that's very unlikely to be a problem. Changed!